src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+    @Inject
+    UserDao userDao;
```

Comment:
```suggestion
    @Inject
    BetDao betDao;
    
    @Inject
    UserDao userDao;
```

Reply:
add access modifiers. Extra line between fields is not necessary